content/core-features/history.textile

```diff
@@ -17,3 +17,19 @@ h2(#persisted-history). Enabling persistent history
 Every message that is persisted to or retrieved from disk counts as an extra message towards your monthly quota. For example, for a channel that has persistence enabled, if a message is published, two messages will be deducted from your monthly quota. If the message is later retrieved from history, another message will be deducted from your monthly quota.
 
 To enable history on a channel, it is necessary to add a channel rule in the settings of your "application dashboard":https://faqs.ably.com/how-do-i-access-my-app-dashboard. See the "documentation on channel rules":https://faqs.ably.com/what-are-channel-rules-and-how-can-i-use-them-in-my-app for further information on what they are and how to configure them.
+
+h2(#ordering). Ordering of historical messages
+
+The order in which historical messages are returned by the History API is based on the message timestamp that was assigned by the channel in the region that the message was published in. This ordering is what Ably calls the canonical global order.
+
+It is important to note that this is not necessarily the order that messages were received by a realtime client, which depends on which region the client is in. 
```

Comment:
I'd maybe suggest splitting this into two sentences to make it simpler to understand:

>It is important to note that this is not necessarily the order that messages were received by a realtime client. The order that each realtime client receives a message depends on which region the client is in.

Reply:
Agree